src/background/extension.ts

```diff
@@ -304,7 +304,7 @@ export class Extension {
     }
 
     toggleURL(url: string) {
-        const isInDarkList = isURLInList(url, this.config.DARK_SITES);
+        const isInDarkList = isURLInList(url, this.config.DARK_SITES) && !this.user.settings.applyToListedOnly;
```

Comment:
`isInDarkList` says that the website is in dark list. Add a second variable if another meaning is needed (e.g. `isEnabledByUser`, `shouldBeRemovedFromEnabledList` or something like this).